examples/ExactlyOnceBasic/Program.cs

```diff
@@ -0,0 +1,680 @@
+        }
+
+        private static async Task CreateTopic(IAdminClient adminClient, string name, int partitions)
+        {
+
+            try
+            {
+                await adminClient.CreateTopicsAsync(new List<TopicSpecification>
+                {
+                    new TopicSpecification
+                    {
+                        Name = name,
+                        NumPartitions = partitions
+                    }
+                });
+                Log($"Created topic {name}");
+            }
+            catch (KafkaException)
+            {
+                // Continue if topic already exists
+                Log($"Topic {name} already exists");
+            }
+        }
+
+        /// <summary>
+        ///     Main method.
+        /// </summary>
+        public static async Task Main(string[] args)
+        {
+            CancellationTokenSource cancelled = new CancellationTokenSource();
+            CancellationTokenSource completed = new CancellationTokenSource();
+            Console.CancelKeyPress += (_, e) =>
+            {
+                e.Cancel = true; // prevent the process from terminating.
+                cancelled.Cancel();
+            };
+            AppDomain.CurrentDomain.ProcessExit += (_, e) =>
+            {
+                cancelled.Cancel();
+                completed.Token.WaitHandle.WaitOne();
+            };
+
+            if (args.Length != 2)
+            {
+                Console.WriteLine("Usage:");
+                Console.WriteLine("create_words <transactional-id>");
+                Console.WriteLine("reverse_words <transactional-id>");
+                completed.Cancel();
+                return;
+            }
+
+            var command = args[0];
+            var transactionalId = args[1];
+
+            IConfiguration configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("./appsettings.json")
+                .Build();
+
+            var producerConfig = configuration.GetSection("Producer").Get<ProducerConfig>().ThrowIfContainsNonUserConfigurable();
+            var consumerConfig = configuration.GetSection("Consumer").Get<ConsumerConfig>().ThrowIfContainsNonUserConfigurable();
+            var generalConfig = configuration.GetSection("General");
+            var wordsConfig = generalConfig.GetSection("Words");
+            var reversedWordsConfig = generalConfig.GetSection("ReversedWords");
+
+            var wordsTopicName = wordsConfig.GetValue<string>("TopicName");
+            var wordsTopicPartitions = wordsConfig.GetValue<int>("TopicPartitions");
+
+            var reversedWordsTopicName = reversedWordsConfig.GetValue<string>("TopicName");
+            var reversedWordsTopicPartitions = reversedWordsConfig.GetValue<int>("TopicPartitions");
+
+            var adminClient = new AdminClientBuilder(producerConfig).Build();
```

Comment:
It would be more proper to have separate admin client config section, even though you can just reuse the producerConfig.